### PR TITLE
[14.0] mig report qweb pdf watermark

### PR DESCRIPTION
**Pull request by fshah-initos**

### CODE REVIEW

**Review by StefanRijnhart — APPROVED** [review]
Thanks!

**Review by CasVissers-360ERP — APPROVED** [review]
Functional test done on migrated Odoo env.

**Comment by StefanRijnhart** [comment]
Oh, needs a rebase now because of changes in .travis.yml.

**Comment by fshah-initos** [comment]
@all can anyone please merge this PR?

File: report_qweb_pdf_watermark/readme/HISTORY.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+12.0.1.0.0 (2019-11-18)
```
Comment (petrus-v) [inline report_qweb_pdf_watermark/readme/HISTORY.rst:1]: this history file differ from the README.rst, not sure if there are some automation to sync those information ?

**Review by petrus-v — APPROVED** [review]
LGTM code review, only notice a detail in change log that could be nice to synchronized between README and HISTORY files.

I'm thinking to improve this module in order to manage a case where each documents would produce a different watermark :thinking:... but that is an other topic !

File: report_qweb_pdf_watermark/readme/HISTORY.rst (hunk 1 above)
Comment (StefanRijnhart) [reply report_qweb_pdf_watermark/readme/HISTORY.rst:1]: Good catch! It's this file that needs to be updated, and then the README.rst will be regenerated automatically.

**Review by StefanRijnhart — CHANGES_REQUESTED** [review]
See Pierre's comment about the history

**Comment by fshah-initos** [comment]
@alexis-via Will you please merge this PR as it is approved?

**Comment by StefanRijnhart** [comment]
@fshah-initos what do you make of the comment about HISTORY.rst?

**Comment by fshah-initos** [comment]
> @fshah-initos what do you make of the comment about HISTORY.rst?

I have updated the history file.

**Comment by fshah-initos** [comment]
@all It would be ncie if someone could merge the PR.
Thank you in advance!

**Review by StefanRijnhart — APPROVED** [review]
Thanks for the update!

**Pull request merged**